src/Provider/Provider.Web/ViewModels/Reports/ProviderApplicationsReport/ProviderApplicationsReportCreateEditModel.cs

```diff
@@ -1,7 +1,12 @@
-﻿namespace Esfa.Recruit.Provider.Web.ViewModels.Reports.ProviderApplicationsReport
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace Esfa.Recruit.Provider.Web.ViewModels.Reports.ProviderApplicationsReport
 {
     public class ProviderApplicationsReportCreateEditModel
     {
+        [FromRoute]
```

Comment:
This should be derived from User claims

Reply:
I'm happy with this. In future we could have a god user that can navigate between providers so we should let the route dictate which provider the user is targeting. So I think decoupling resource identification from authorisation is a good thing.

The authorisation filter is triggered on every page request so I know I can trust this ukprn.

Reply:
While I am not sure what the "god" user requirements are, I am happy that authorisation filter is safe guarding the value in URL.